.pre-commit-config.yaml

```diff
@@ -1,11 +1,15 @@
 # .pre-commit-config.yaml
 repos:
--   repo: https://github.com/awslabs/cfn-python-lint
-    rev: v0.61.1 # The version of cfn-lint to use
-    hooks:
-    -   id: cfn-python-lint
-        files: zs_cc_cf_template_.*\.(json|yml|yaml)$
--   repo: https://github.com/jorisroovers/gitlint
-    rev:  v0.17.0
-    hooks:
-    -   id: gitlint
\ No newline at end of file
+  - repo: https://github.com/awslabs/cfn-python-lint
+      rev: v0.61.1 # The version of cfn-lint to use
+      hooks:
+      -   id: cfn-python-lint
+          files: zs_cc_cf_template_.*\.(json|yml|yaml)$
+  - repo: https://github.com/jorisroovers/gitlint
+      rev:  v0.17.0
+      hooks:
+      -   id: gitlint
+  - repo: https://github.com/Yelp/detect-secrets
```

Comment:
Were you able to run this succesfully against local repo? Can we add what the output looks like/supposed to look like and what are failure mitigaition methods?